web/annotation_editor_layer_builder.js

```diff
@@ -67,14 +67,17 @@ class AnnotationEditorLayerBuilder {
     const clonedViewport = viewport.clone({ dontFlip: true });
     if (this.div) {
       this.annotationEditorLayer.update({ viewport: clonedViewport });
-      this.show();
+      if (!this.annotationEditorLayer.isEmpty()) {
+        this.show();
+      }
```

Comment:
How about moving the `isEmpty` check into the `show`-method instead and just call it unconditionally here, since that seems "safer" with any future changes?